index.html

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+---
+
+<div class="top-strip"></div>
+<div class="content">
+  <main class="container">
+  	<div class="row-fluid">
+  		<section class="content">
+  		  <h1>ICM Quizzes</h1>
+  		</section>
+  		<ul class="entries">
+  		{% for post in site.posts %}
+  		  <li>
+  		    <a href="{{ post.url | prepend: site.baseurl }}">
+		        <h3>{{ post.title }}</h3>
```

Comment:
great to have this automated but it's just going to be a few quizzes so can hard-code navigation.